components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1235,6 +1235,43 @@ const string kDegenChainId = "0x27bc86aa";
 const string kLineaChainId = "0xe708";
 const string kOptimismSepoliaChainId = "0xaa37dc";
 const string kZoraChainId = "0x76adf1";
+const string kAstarChainId = "0x250";
+const string kAstarZkEVMChainId = "0xec0";
+const string kBahamutChainId = "0x142d";
+const string kBitTorrentChainMainnetChainId = "0xc7";
+const string kCoreChainId = "0x45c";
+const string kCronosMainnetChainId = "0x19";
+const string kCyberMainnetChainId = "0x1d88";
+const string kElysiumMainnetChainId = "0x53b";
+const string kEOSEVMNetworkChainId = "0x4571";
+const string kHederaMainnetChainId = "0x127";
+const string kImmutableZkEVMChainId = "0x343b";
+const string kKlaytnMainnetCypressChainId = "0x2019";
+const string kKromaChainId = "0xff";
+const string kLightlinkPhoenixMainnetChainId = "0x762";
+const string kMoonbeamChainId = "0x504";
+const string kMoonriverChainId = "0x505";
+const string kNearMainnetChainId = "0x18d";
+const string kOasisEmeraldChainId = "0xa516";
+const string kOasisSapphireChainId = "0x5afe";
+const string kOasysMainnetChainId = "0xf8";
+const string kOntologyMainnetChainId = "0x3a";
+const string kOpBNBMainnetChainId = "0xcc";
+const string kRangersProtocolMainnetChainId = "0x7e9";
+const string kRoninChainId = "0x7e4";
+const string kShidenChainId = "0x150";
+const string kSkaleCalypsoHubChainId = "0x5d456c62";
+const string kSkaleEuropaHubChainId = "0x79f99296";
+const string kSkaleNebulaHubChainId = "0x585eb4b1";
+const string kSkaleTitanHubChainId = "0x507aaa2a";
+const string kTelosEVMMainnetChainId = "0x28";
+const string kThetaMainnetChainId = "0x169";
+const string kThunderCoreMainnetChainId = "0x6c";
+const string kWemixMainnetChainId = "0x457";
+const string kXaiMainnetChainId = "0xa1337";
+const string kZetaChainMainnetChainId = "0x1b58";
+const string kZilliqaEVMChainId = "0x8001";
+const string kChainId = "";
```

Comment:
why do we need a empty chain id?

Reply:
We don't, thanks for catching this. Removed here: [05f5608](https://github.com/brave/brave-core/pull/23297/commits/05f56083b6a55fabd8fbffc0449563830c31eba3)